appserver/pom.xml

```diff
@@ -303,9 +303,9 @@
                 <version>${openmq.version}</version>
             </dependency>
             <dependency>
-                <groupId>com.sun.messaging.mq</groupId>
-                <artifactId>imqjmx</artifactId>
-                <version>4.6-b01</version>
+                <groupId>org.glassfish.mq</groupId>
```

Comment:
- I knew it looked familiar: https://github.com/eclipse-ee4j/glassfish/pull/23881/files#r840627381

Reply:
Wait a minute, is your change showing the replacement? Excellent!

Reply:
> Wait a minute, is your change showing the replacement?

At least partial. That's great!

Reply:
Haha, I noticed it today, first I have found the 5.1, then Payara's special releases, then I opened the original jar, found some class and asked GitHub to find it.
I hope I will get to JMX next year, it needs some care as parts are still somehow in the year of 2000.